controllers/platform/platformutils.go

```diff
@@ -45,43 +47,43 @@ func ConfigureRegistry(ctx context.Context, c client.Client, p *operatorapi.Sona
 	//@TODO Add a notification on the status about this registry value ignored when https://issues.redhat.com/browse/KOGITO-9218 will be implemented
 	if p.Spec.BuildPlatform.BuildStrategy == operatorapi.PlatformBuildStrategy && p.Status.Cluster == operatorapi.PlatformClusterOpenShift {
 		p.Spec.BuildPlatform.Registry = operatorapi.RegistrySpec{}
-		log.Info("Platform registry not set and ignored on openshift cluster")
+		klog.V(log.I).InfoS("Platform registry not set and ignored on openshift cluster")
 		return nil
 	}
 
 	if p.Spec.BuildPlatform.Registry.Address == "" && p.Status.Cluster == operatorapi.PlatformClusterKubernetes {
 		// try KEP-1755
 		address, err := GetRegistryAddress(ctx, c)
 		if err != nil && verbose {
-			log.Error(err, "Cannot find a registry where to push images via KEP-1755")
+			klog.V(log.E).ErrorS(err, "Cannot find a registry where to push images via KEP-1755")
 		} else if err == nil && address != nil {
 			p.Spec.BuildPlatform.Registry.Address = *address
 		}
 	}
 
-	log.Debugf("Final Registry Address: %s", p.Spec.BuildPlatform.Registry.Address)
+	klog.V(log.D).InfoS("Final Registry Address", "address", p.Spec.BuildPlatform.Registry.Address)
 	return nil
 }
 
 func SetPlatformDefaults(p *operatorapi.SonataFlowPlatform, verbose bool) error {
 	if p.Spec.BuildPlatform.BuildStrategyOptions == nil {
-		log.Debugf("SonataFlow Platform [%s]: setting publish strategy options", p.Namespace)
+		klog.V(log.D).InfoS("SonataFlow Platform: setting publish strategy options", "namespace", p.Namespace)
 		p.Spec.BuildPlatform.BuildStrategyOptions = map[string]string{}
 	}
 
 	if p.Spec.BuildPlatform.GetTimeout().Duration != 0 {
 		d := p.Spec.BuildPlatform.GetTimeout().Duration.Truncate(time.Second)
 
 		if verbose && p.Spec.BuildPlatform.Timeout.Duration != d {
-			log.Log.Infof("ContainerBuild timeout minimum unit is sec (configured: %s, truncated: %s)", p.Spec.BuildPlatform.GetTimeout().Duration, d)
+			klog.V(log.I).InfoS("ContainerBuild timeout minimum unit is sec", "configured", p.Spec.BuildPlatform.GetTimeout().Duration, "truncated", d)
 		}
 
-		log.Debugf("SonataFlow Platform [%s]: setting build timeout", p.Namespace)
+		klog.V(log.D).InfoS("SonataFlow Platform: setting build timeout", "namespace", p.Namespace)
 		p.Spec.BuildPlatform.Timeout = &metav1.Duration{
 			Duration: d,
 		}
 	} else {
-		log.Debugf("SonataFlow Platform [%s]: setting default build timeout to 5 minutes", p.Namespace)
+		klog.V(log.D).InfoS("SonataFlow Platform: setting default build timeout to 5 minutes", "namespace", p.Namespace)
```

Comment:
Can be used Infof to have the placeholder  "SonataFlow Platform [namespace]:" ?

Reply:
No, as we loose the structured logging @ricardozanini wanted to introduce.

Reply:
@manstis ok, but remove the ":"